.travis.yml

```diff
@@ -1,32 +1,25 @@
 language: cpp
 
+addons:
+  apt:
+    packages:
+      - cmake 
+      - libsdl2-dev
+      - libsdl2-net-dev
+      - rpm
+
 install:
-  # Download and build premake5 from source; the Travis environment doesn't have the right version of glibc6 for the prebuilt binaries to work.
-  - wget https://github.com/premake/premake-core/releases/download/v5.0.0-alpha6/premake-5.0.0-alpha6-src.zip -O premake.zip
-  - unzip premake.zip
-  - cd premake-5.0.0-alpha6/build/gmake.unix
-  - make config=release
-  - cd ../../..
-  - mv premake-5.0.0-alpha6/bin/release/premake5 premake5
-  # Download and build both SDL2 from source
-  - wget http://libsdl.org/release/SDL2-2.0.9.tar.gz
-  - tar -xvf SDL2-2.0.9.tar.gz
-  - pushd SDL2-2.0.9/
-  - ./configure --prefix=/usr && make && sudo make install
-  - popd
-  # Run sdl2-config for debugging -- protentially remove this
-  - sdl2-config --cflags --libs
-  # Download and build both SDL2_net from source
-  - wget https://www.libsdl.org/projects/SDL_net/release/SDL2_net-2.0.1.tar.gz
-  - tar -xvf SDL2_net-2.0.1.tar.gz
-  - pushd SDL2_net-2.0.1/
-  - ./configure --prefix=/usr && make && sudo make install
-  - popd
 
 
 # Run premake to generate makefiles.
 before_script:
-  - ./premake5 gmake
+  - mkdir build
+  - pushd build
+  - cmake .. -DCMAKE_INSTALL_PREFIX=./install
+  - popd
 
 script:
-  - make config=release
+  - pushd build
+  - make install
```

Comment:
Can this build in parallel (`-j`) to speed up the Travis run?

Reply:
Good Idea, it certainly makes my build faster.